go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/sirupsen/logrus v1.7.0
 	github.com/spf13/viper v1.7.1
 	github.com/urfave/cli/v2 v2.3.0
+	go.uber.org/automaxprocs v1.4.0
```

Comment:
As this is a new runtime dependency, we need you to update the LICENSE.

Reply:
Done